test/testing/test_pytest_plugin.py

```diff
@@ -40,60 +40,145 @@ def test_rabbit_manager(rabbit_manager):
     assert "/" in [vhost['name'] for vhost in rabbit_manager.get_all_vhosts()]
 
 
-def test_rabbit_config_leftover_connections(testdir, plugin_options):
-
-    # run a test that leaves connections lying around
-    testdir.makepyfile(
-        """
-        from nameko.containers import ServiceContainer
-        from nameko.rpc import rpc
-
-        class Service(object):
-            name = "service"
-
-            @rpc
-            def method(self):
+class TestFastTeardown(object):
+
+    def test_order(self, testdir):
+
+        testdir.makeconftest(
+            """
```

Comment:
we might want to think about "hosting" these tests in actual files (maybe outside the main test tree) so they don't have to live inside docstrings